pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -27,11 +27,11 @@ rec {
   stable = if stdenv.hostPlatform.system == "i686-linux" then legacy_390 else latest;
 
   production = generic {
```

Comment:
This is a "new feature branch" release, so you need to update the `latest` package, not `production`